central/scannerdefinitions/handler/handler.go

```diff
@@ -654,3 +660,32 @@ func getOfflineFileVersion(mf *os.File) (string, error) {
 	}
 	return m.Version, nil
 }
+
+func validateV4DefsVersion(zipPath string) error {
+	zipR, err := zip.OpenReader(zipPath)
+	if err != nil {
+		return errors.Wrap(err, "couldn't open file as zip")
+	}
+	defer utils.IgnoreError(zipR.Close)
+
+	for _, zipF := range zipR.File {
+		if strings.HasPrefix(zipF.Name, scannerV4DefsPrefix) {
+			mf, err := openFromArchive(zipF.Name, "manifest.json")
+			if err != nil {
+				return errors.Wrap(err, "couldn't open v4 defs manifest.json")
+			}
+
+			offlineV, err := getOfflineFileVersion(mf)
+			if err != nil {
+				return errors.Wrap(err, "couldn't read v4 defs manifest.json")
+			}
+			utils.IgnoreError(mf.Close)
+			if offlineV != "dev" && offlineV != minorVersionPattern.FindString(version.GetMainVersion()) {
+				msg := fmt.Sprintf("failed to upload offline vuln file, uploaded file is version: %s and system version is: %s", offlineV, version.GetMainVersion())
```

Comment:
What is the official name of the bundle? Offline bundle? You may say that "Offline bundle version x is not supported. The system is running in x.y.z and please upload an offline bundle version x.y"
You may also recommend the command "roxctl scanner download-db" here.